bool save_map
string map_name
---
bool success
string status_message